larpandoracontent/LArMonitoring/HierarchyValidationAlgorithm.cc

```diff
@@ -433,7 +489,8 @@ void HierarchyValidationAlgorithm::MCValidation(const LArHierarchyHelper::MatchI
                     {
                         try
                         {
-                            const CartesianVector recoVertex{LArPfoHelper::GetVertex(pRecoNode->GetLeadingPfo())->GetPosition()};
+                            const CartesianVector recoVertex{isSignal == 1 ? 
+				    LArPfoHelper::GetVertex(pRecoNode->GetLeadingPfo())->GetPosition() : null};
```

Comment:
I think this should probably be on one line - clang format will handle the formatting.